core/src/main/kotlin/fr/sncf/osrd/api/api_v2/conflicts/ConflictDetectionRequest.kt

```diff
@@ -11,20 +11,46 @@ import fr.sncf.osrd.utils.json.UnitAdapterFactory
 import java.time.ZonedDateTime
 
 class ConflictDetectionRequest(
+    /** Infra ID. */
     var infra: String,
+    /** Expected infra version. */
```

Comment:
Maybe would be useful to add the purpose here. For instance, "Used for cache invalidation."?

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/9391/commits/649f0b097ad6f6412d4e5a8a1842faef22ee3e70